### PR TITLE
Changes to address multiple issues

### PR DESCRIPTION
**Pull request by DavidMcCann-NOC**

### CODE REVIEW

**Comment by DavidMcCann-NOC** [comment]
I've also added my environment.yml file for re-building the env. Notebooks were added to save to my branch before testing the rebasing problem at my end

File: env/environment.yml [hunk 1]
```diff
@@ -14,5 +14,6 @@ dependencies:
   - spyder=3.3.6
   - sphinx=4.4.0
   - sphinx_rtd_theme=1.0.0
-  - xarray=0.20.1
+  - xarray=2022.11.0
```
Comment (ACHMartin) [inline env/environment.yml:17]: we should probably add `datetime` as well even if I think it is a dependence in `xarray`

File: seastar/oscar/level1.py [hunk 2]
```diff
@@ -40,24 +40,13 @@ def fill_missing_variables(ds_dict, antenna_id):
...
-                    data=np.full(ds_dict[mid_id][var].shape, np.NaN),
-                    coords=ds_dict[mid_id][var].coords,
-                    dims=ds_dict[mid_id][var].dims)
+    antenna_list = [fore_id, mid_id, aft_id]
+
+    for antenna_1 in antenna_list:
+        for antenna_2 in [a for a in antenna_list if a not in [antenna_1]]:
+            for var in ds_dict[antenna_1].data_vars:
+                if var not in ds_dict[antenna_2].data_vars:
+                    ds_dict[mid_id][var] = xr.DataArray(data=np.NaN)
```
Comment (ACHMartin) [inline seastar/oscar/level1.py:49]: much shorter version indeed :).
Do we need to specify line 40-42 the antennas (Fore, Aft, Mid)?

Line 49: should it be ds_dict[ANTENNA_2][var] = xr...Nan

**Review by ACHMartin — CHANGES_REQUESTED** [review]
ready to go after the modifications

File: config.ini [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # Device names are in [] brackets and are generated using the platform
 # module and platform.node(). 
 [LIVPCZG-10]
-oscar_path = D:\data\SEASTAR\SEASTARex\Data\Metasensing\OSCAR\Brest_Sample_Dec_8\
+oscar_path = D:\data\SEASTAR\SEASTARex\Data\Metasensing\OSCAR\
```
[the remark this answers is not in the document]
Comment (DavidMcCann-NOC) [reply config.ini:5]: Ha! A typo on my system's folder name no less...thanks!

File: env/environment.yml (hunk 1 above)
Comment (DavidMcCann-NOC) [reply env/environment.yml:17]: Good idea

File: seastar/oscar/level1.py (hunk 2 above)
Comment (DavidMcCann-NOC) [reply seastar/oscar/level1.py:49]: Good catch

**Pull request merged**